rgw/v2/tests/s3cmd/test_bucket_stats.py

```diff
@@ -19,6 +19,7 @@
 import json
 import logging
 import os
+import subprocess
```

Comment:
@anrao19 , could you remove the exception raised in the utils.exec_shell_cmd method, instead of calling the sub_process in the script file

Reply:
@mkasturi18 Made changes